### ISSUE TITLE
Named entity score.

### ISSUE THREAD

**Issue by ChunchuanLv** [sheng-z/stog#2]
Hi Sheng,

Thanks for your nice work. The conversion to dependency parsing is really insightful. 

One question, I find your named entity metric to be low, and yet have a wikification accuracy higher than that of named entity score. Do you know what is happening? Are you having low named entity recall or there are issues with surface string processing?

Chunchuan

**Comment by sheng-z** [comment]
Hi Chunchuan,

The NER score is lower than yours, because the NER step (i.e. ``"United States" -> country``) is done in preprocessing, which are just simple rules observed based on the training data.

Wikification in contrast is done by the powerful DBpedia Spotlight API. It directly works on the input text, and is therefore not impacted by NER. I think this is the reason why the wikification score is higher than NER. 

Replacing my current rule-based NER with a SOTA NER tagger would probably further improve the final result.

**Comment by ChunchuanLv** [comment]
Thank you for your quick response. Another question, is there any technical
reason that you don't use AllenNLP  as it is? I'd like to adopt your code
to pytorch 1.0 and the updated AllenNLP, Bert implementation. Do you think
there will be any problem?

Chunchuan

**Comment by sheng-z** [comment]
No technical reason. There was just some internal story behind it. 
Anyway, moving it into AllenNLP is completely doable and appreciated ;)

**Comment by ChunchuanLv** [comment]
Cool, thanks.

Chunchuan
